Common/Core/RecoDecay.h

```diff
@@ -24,6 +24,8 @@
 #include <vector>    // std::vector
 
 #include "CommonConstants/MathConstants.h"
+#include "Framework/AnalysisDataModel.h"
+#include "TMCProcess.h" // for VMC Particle Production Process
```

Comment:
There is no file `TMCProcess.h` in O2 nor in O2Physics.

Reply:
The file is in VMC. It could be fine it does not add a dependency on any linked library.

Reply:
Okay, so can I leave line 28 as it is now?

Reply:
Sorry, I missed "if" in my previous message. I meant: "It could be fine if it does not add a dependency on any linked library."